app/_hub/kong-inc/vault-auth/index.md

```diff
@@ -108,25 +108,27 @@ A Vault object represents the connection between Kong and a Vault server. It def
 Vault objects can be created via the following HTTP request:
 
 ```bash
-$ curl -X POST http://kong:8001/vaults \
-  --data name=kong-auth \
-  --data mount=kong-auth \
-  --data protocol=http \
-  --data host=127.0.0.1 \
-  --data port=8200
-  --data token=s.m3w9gdV0uMDYFpMgEWSB2mtM
+$ curl --request POST \
+  --url http://localhost:8001/vaults \
```

Comment:
```suggestion
$ curl -X POST http://localhost:8001/vaults \
```
`-X` is the shorthand for `--request`. Just changing this to the standard style we tend to use across the docs (tested, this does work).